lib/src/git_backend.rs

```diff
@@ -376,7 +382,18 @@ fn commit_from_git_without_root_parent(
     let author = signature_from_git(commit.author());
     let committer = signature_from_git(commit.committer());
 
-    Commit {
+    // Re-parse the commit data to extract the signature and the signed data
+    // (which is the commit buffer with the gpgsig header omitted)
+    // Ideally, we could use try_to_commit_ref_iter at the beginning of this
+    // function For now, this works
```

Comment:
nit: missing period

Reply:
It was on the next line and then `cargo fmt` got it :)